db/deploy/util_functions/upsert_archive_trigger.sql

```diff
@@ -0,0 +1,53 @@
+-- Deploy ccbc:util_functions/upsert_archive_trigger to pg
+-- requires: trigger_function/set_archived
+
+begin;
+
+create or replace function ccbc_private.upsert_archive_trigger(
+  table_schema_name text,
+  table_name text,
+  user_table_schema_name text default 'ccbc_public',
+  user_table_name text default 'ccbc_user'
```

Comment:
I notice `user_table_schema_name` and  `user_table_name` isn't used, should probably delete it but won't block approval

Reply:
indeed  user_table_schema_name and user_table_name isn't used. I deleted them.